CONTRIBUTING.md

```diff
@@ -6,9 +6,9 @@ one should follow.
 
 ## Code of Conduct
 
-The code of conduct ("CoC") can be found in the file "COC.md". Everybody
-participating in this project must adhere to the rules and guidelines stated in
-this CoC.
+The code of conduct ("CoC") can be found in the file {file:COC "COC.md"}.
+Everybody participating in this project must adhere to the rules and
+guidelines stated in this CoC.
```

Comment:
This isn't needed. To be honest I'm kinda annoyed by YARD's hyperlinks (since they're non-standard) so I'd rather just keep the COC Markdown as-is.